.github/ISSUE_TEMPLATE/2-feature_request.yml

```diff
@@ -19,17 +19,15 @@
 
 name: Feature request
 description: Suggest an idea for this project
-title: "[FEATURE REQUEST]"
-labels: enhancement
+title: "[FEATURE REQUEST] (provide short, concise summary)"
```

Comment:
Same here for feature request in the title as we have the label. It could be a design proposal too.